js/guides.js

```diff
@@ -133,7 +133,18 @@ function postProcessCodeBlocks() {
     //  3) There is exactly 1 small set of languages to choose from. This does not allow for multiple language preferences. For example, users cannot prefer both Kotlin and ZSH.
     //  4) Only 1 sample of each language can exist in the same collection.
 
-    var preferredBuildScriptLanguage = window.localStorage.getItem("preferred-gradle-dsl") || "groovy";
+    var GRADLE_DSLs = ["groovy", "kotlin"];
```

Comment:
Technically, it could be. Browsers don't actually enforce immutability IIUC, so it'd be kind of a marker. We could use `const` a _lot_ more places if it actually made things immutable. I'd care more if we were maintaining more than 1 file or > 4000 lines of JS.